src/components/graph-view.js

```diff
@@ -199,7 +199,7 @@ class GraphView extends Component {
       .on('keydown', null);
   }
 
-  componentWillReceiveProps(nextProps) {
+  UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
Don't change this yet. While we are working toward supporting 16.3 natively, we still run on React 15 in the 4.x versions. The tests will use React 16 to make sure we're compatible with newer projects.

The next major version will require React 16.3+